app/components/Link.tsx

```diff
@@ -1,9 +1,9 @@
 import {
   Link as RemixLink,
-  type LinkProps as RemixLinkProps,
   NavLink as RemixNavLink,
-  type NavLinkProps as RemixNavLinkProps,
   useMatches,
+  type LinkProps as RemixLinkProps,
```

Comment:
TypeScript LSP wants the types to imported last like this. 

```tsx
import {User, type UserProps} from 'dep';
```

The type should always be last.